website/data/docs/api/comparisons/compare.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: compare
+description: Compares the value of a Dinero object relative to another.
+returns: number
+---
+
+Compares the value of a Dinero object relative to another; useful for sorting Dinero objects.
+
+Returns a negative number if the Dinero object is less than the other; 0 if they are equal; or a positive number if greater than the other.
+
+**You can only compare objects that share the same currency.** The function also normalizes objects to the same scale (the highest) before comparing them.
+
+## Parameters
+
+<Parameters>
+
+<Parameter name="dineroObject" type="Dinero<TAmount>" required={true}>
+
+The first Dinero object to compare.
+
+</Parameter>
+
+<Parameter name="comparator" type="Dinero<TAmount>" required={true}>
+
+The second Dinero object to compare.
+
+</Parameter>
+
+</Parameters>
+
+## Code examples
+
+### Compare two objects
+
+```js
+import { dinero, compare } from 'dinero.js';
+import { USD } from '@dinero.js/currencies';
+
+const d1 = dinero({ amount: 800, currency: USD });
+const d2 = dinero({ amount: 500, currency: USD });
+
+compare(d1, d2); // 1
+```
+
+### Compare two objects after normalization
+
+```js
+import { dinero, compare } from 'dinero.js';
+import { USD } from '@dinero.js/currencies';
+
+const d1 = dinero({ amount: 5000, currency: USD, scale: 3 });
+const d2 = dinero({ amount: 800, currency: USD });
+
+compare(d1, d2); // -1
+
+const d3 = dinero({ amount: 5000, currency: USD, scale: 3 });
+const d4 = dinero({ amount: 500, currency: USD });
+
+compare(d3, d4); // 0
+```
```

Comment:
Since this is one of the main usages, you can also add an example to show how to sort Dinero objects with this function.